js/scenes/Home.js

```diff
@@ -144,16 +154,26 @@ const styles = StyleSheet.create({
     ...defaultTextStyle,
   },
 
-  centering: {
+  loading: {
     alignItems: 'center',
     justifyContent: 'center',
     padding: 8,
   },
 });
 
-const mapStateToProps = ({ items }) => {
+const mapStateToProps = ({ items, itemsCache }) => {
+  const itemIds = items.value || [];
+
+  // create a list of fetched/inflated items
+  const fetchedItems = itemIds.reduce((result, id) => {
```

Comment:
A filter would just just return a list of ids, but actually want to inflate the list into fetched item objects. If items aren't in the `itemsCache` yet, I don't want to render them. I guess I could do `.filter()` then `.map()`?